### PR TITLE
Added a javascript question based on loop through arrays

### PR DESCRIPTION
**Pull request by robinathon**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In JavaScript, Which function of an Array object calls a function for each element in the array?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:4]:
```suggestion
      "In JavaScript, which method uses a callback function for each element in the array?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    Answer: "forEach()",
+    Distractor1: "every(",
+    Distractor2: "forEvery()",
+    Distractor3: "each()",
+    Explanation:
+      "forEach() – Calls a function for each element in the array.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:10]:
```suggestion
      "The forEach() method uses a callback function for each element in the array.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by robinathon** [comment]
I made the adjustments.
ThankYou

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**